### DISCUSSION TITLE
Emoji in prompt looks smaller

### DISCUSSION THREAD

**Discussion by azzamsa** [kovidgoyal/kitty#7689] [Q&A] [answered]
Hi,

I use this config in my starship:

```toml
[shell]
disabled = false
fish_indicator = '🐡'
nu_indicator = '🦀'
```

Wezterm had better sizing than kitty in this regard.
The emoji in kitty is smaller than the rest of the font.
Any way to fix this?

Kitty:
![image](https://github.com/user-attachments/assets/09053352-3b6a-48d7-b169-402d206566c5)

Wezterm:
![image](https://github.com/user-attachments/assets/eef1c856-e63d-47cf-9585-c72b0bffc644)

```
---
OS: Fedora Linux 40 (Workstation Edition) x86_64
DE: GNOME 46.3.1
WM: Mutter (Wayland)

dotfiles on  master [✘!?] 
🐡 ❯ kitty --version
kitty 0.35.2 created by Kovid Goyal

dotfiles on  master [✘!?] 
🐡 ❯ wezterm --version
wezterm 20240203-110809-5046fc22
```

**Comment by kovidgoyal** [comment]
Both WezTerm and kitty are correctly rendering the symbol in exactly two full cells as per your
screenshot. So not sure what you are asking. The aspect ratio of the
cells are different in the two terminal emulators so the one in WezTerm
is slightly larger.

**Reply by azzamsa** [reply]
> So not sure what you are asking

How to make the emoji in kitty the same size as the `>` char (as in Wezterm)?

**Comment by kovidgoyal** [comment, marked as answer]
On Fri, Aug 02, 2024 at 07:51:46PM -0700, Azzam S.A wrote:
> > So not sure what you are asking
>
> How to make the emoji in kitty the same size as the `>` char (as in Wezterm)?

Use modify_font to make the cells wider.
